src/consts.rs

```diff
@@ -170,11 +170,9 @@ impl<'gcc, 'tcx> CodegenCx<'gcc, 'tcx> {
             match kind {
                 Some(kind) if !self.tcx.sess.fewer_names() => {
                     let name = self.generate_local_symbol_name(kind);
-                    // TODO(antoyo): check if it's okay that TLS is off here.
```

Comment:
Only statics can be thread local. Allocations referenced by other allocations can't ever be thread local as a single pointer has to be used independent of the thread.